fdbclient/include/fdbclient/Tenant.h

```diff
@@ -232,6 +228,9 @@ struct TenantMetadata {
 	static inline auto& tenantGroupTenantIndex() { return instance().tenantGroupTenantIndex; }
 	static inline auto& tenantGroupMap() { return instance().tenantGroupMap; }
 	static inline auto& lastTenantModification() { return instance().lastTenantModification; }
+	// This system keys stores the tenant id prefix that is used during metacluster/standalone cluster creation. If the
+	// key is not present then we will assume the prefix to be 0
+	static KeyBackedProperty<int64_t>& tenantIdPrefix();
```

Comment:
Good idea putting it here, since this won't be part of the metacluster's tenant metadata